aws/email.go

```diff
@@ -0,0 +1,75 @@
+package aws
+
+import (
+	"bytes"
+	encsv "encoding/csv"
+	amaws "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ses"
+	c "github.com/companieshouse/innovation-accounts-statistics-tool/config"
+	"github.com/companieshouse/innovation-accounts-statistics-tool/models"
+	"gopkg.in/gomail.v2"
+	"io"
+)
+
+const (
+	subject   = "Small Full Accounts Statistics"
+	body      = "<h1>SFA Stats</h1><p>Attached is the CSV of the stats</p>"
+	awsRegion = "eu-west-1"
+)
+
+// EmailGenerator provides an interface by which to interact with aws emails.
+type EmailGenerator interface {
+	GenerateEmail(csv *models.CSV, cfg *c.Config) error
+}
+
+// Impl is a concrete implementation of the EmailGenerator interface.
+type Impl struct {
```

Comment:
Shall we add a member variable to this struct to hold config, similar to what we do for service? That way, the `GenerateEmail` method exposed by the interface can become tidier